src/Definition/Reflection/ClassDefinition.php

```diff
@@ -83,11 +80,6 @@ private function reflectParameters(): void
             $parameter                               = new Parameter($parameterReflection);
             $this->parameters[$parameter->getName()] = $parameter;
         }
-
-        uasort(
```

Comment:
@sjokkateer can we please make sure that your test is also committed with these changes?

Reply:
Yes, does this suffice? It is however a test for demonstrative purposes though, the real assertions are done by existing tests I assume, that resolve objects.

Reply:
This is more than enough, thanks!